### DISCUSSION TITLE
Been getting an error that says the source code violates salesforce rules and policies

### DISCUSSION THREAD

**Discussion by dulzorigo** [thelovekesh/ghost-v4-on-heroku#24] [Q&A] [answered]
Hi, 

Been getting an error that says the source code violates salesforce rules and policies. Is this happening to only me or is everyone experiencing the same issue?

Thanks

**Comment by thelovekesh** [comment, marked as answer]
Try fork and then rename the repo. You will be able to deploy your Ghost app.
